tests/fixtures/fixtures_ro_crate.py

```diff
@@ -0,0 +1,234 @@
+# pylint: disable=missing-docstring
+# nosec assert_used
+import copy
+import json
+import uuid
+from pathlib import Path
+
+import pytest
+
+from src.blueprints.datafile import RawDatafile
+from src.blueprints.dataset import RawDataset
+from src.blueprints.experiment import RawExperiment
+from src.blueprints.project import RawProject
+from src.profiles.ro_crate.ro_crate_parser import ROCrateParser
+
+
+@pytest.fixture()
+def fixture_rocrate_uuid() -> str:
+    return uuid.uuid4().hex
+
+
+@pytest.fixture()
+def fixture_ro_crate_name() -> str:
+    return "Testing_Ro-crate"
+
+
+@pytest.fixture()
+def fixture_rocrate_project(raw_project: RawProject) -> RawProject:
+    ro_crate_project = copy.copy(raw_project)
+    return ro_crate_project
+
+
+@pytest.fixture()
+def fixture_rocrate_experiment(raw_experiment: RawExperiment) -> RawExperiment:
+    ro_crate_experiment = copy.copy(raw_experiment)
+    return ro_crate_experiment
```

Comment:
If I understand correctly, this fixture is just making a deep copy of the fixture passed in - wondering where the need for that comes from? Or is the intention that the definition of an RO-Crate experiment might change in future and this allows us to change it more easily?

Reply:
Exactly. Same answer as above I thought the RO-Crate fixtures might need to be modified/extended (and they are looking like they might in future) so made copies, also so I didn't modify the fixtures a lot of the other tests rely on.